Package.swift

```diff
@@ -4,7 +4,7 @@ import PackageDescription
 let package = Package(
     name: "CobrowseIO",
     platforms: [
-        .macOS(.v10_13), .iOS(.v11)
+        .macOS(.v10_10), .iOS(.v9)
```

Comment:
We're iOS11+ now https://docs.cobrowse.io/sdk-installation/ios#requirements

Reply:
And macOS 10.13+ https://docs.cobrowse.io/sdk-installation/macos#requirements

Reply:
Reverted.